src/content/scripting/playlist_parser_script.cc

```diff
@@ -42,6 +42,12 @@
 
 #define ONE_TEXTLINE_BYTES 1024
 
+#ifdef __linux__
+static constexpr bool linux = true;
+#else
+static constexpr bool linux = false;
+#endif
+
```

Comment:
What about moving this to a header like `compat.h`. Avoids having the same code in many files

Reply:
Fixed.